skflow/data_feeder.py

```diff
@@ -23,7 +23,6 @@
 import numpy as np
 from sklearn.utils import check_array
 
-
```

Comment:
I think it should be two lines after imports.
https://www.python.org/dev/peps/pep-0008/#blank-lines